static/app/views/detectors/components/forms/metric.tsx

```diff
@@ -0,0 +1,642 @@
+                    ['lower', t('lower')],
+                  ]}
+                />
+                <span>than the previous</span>
+                <StyledSelectField
+                  name="config.low_threshold.unit"
+                  hideLabel
+                  inline
+                  defaultValue="1 hour"
+                  flexibleControlStateSize
+                  choices={[
+                    ['5 minutes', '5 minutes'],
+                    ['15 minutes', '15 minutes'],
+                    ['1 hour', '1 hour'],
+                    ['1 day', '1 day'],
+                    ['1 week', '1 week'],
+                    ['1 month', '1 month'],
+                  ]}
+                />
+              </Flex>
+            </Flex>
+          )}
+          {kind === 'dynamic' && (
+            <Flex column>
+              <SelectField
+                required
+                name="config.sensitivity"
+                label={t('Level of responsiveness')}
+                help={t(
+                  'Choose your level of anomaly responsiveness. Higher thresholds means alerts for most anomalies. Lower thresholds means alerts only for larger ones.'
+                )}
+                defaultValue={AlertRuleSensitivity.MEDIUM}
+                choices={[
+                  [AlertRuleSensitivity.HIGH, t('High')],
+                  [AlertRuleSensitivity.MEDIUM, t('Medium')],
+                  [AlertRuleSensitivity.LOW, t('Low')],
+                ]}
+              />
+              <SelectField
+                required
+                name="config.thresholdType"
+                label={t('Direction of anomaly movement')}
+                help={t(
+                  'Decide if you want to be alerted to anomalies that are moving above, below, or in both directions in relation to your threshold.'
+                )}
+                defaultValue={AlertRuleThresholdType.ABOVE_AND_BELOW}
+                choices={[
+                  [AlertRuleThresholdType.ABOVE, t('Above')],
+                  [AlertRuleThresholdType.ABOVE_AND_BELOW, t('Above and Below')],
+                  [AlertRuleThresholdType.BELOW, t('Below')],
+                ]}
+              />
+            </Flex>
+          )}
+        </Flex>
+      </Section>
+    </Container>
+  );
+}
+
+function OwnerField() {
+  return (
+    <StyledMemberTeamSelectorField
+      placeholder={t('Select a member or team')}
+      label={t('Owner')}
+      help={t('Sentry will assign new issues to this owner.')}
+      name="owner"
+      flexibleControlStateSize
+    />
+  );
+}
+
+const FormStack = styled(Flex)`
+  max-width: ${p => p.theme.breakpoints.xlarge};
+  flex-direction: column;
+  gap: ${space(4)};
+  padding: ${space(4)};
+`;
+
+const FirstRow = styled('div')`
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  gap: ${space(1)};
+  border-bottom: 1px solid ${p => p.theme.border};
+`;
+
+const DetectColumn = styled(Flex)`
+  flex-grow: 1;
+`;
```

Comment:
feels like styles are coming from 3 directions here. inline flex, inline styles, emotion, Flex

Reply:
Agreed! Cleaned up in [82efb48](https://github.com/getsentry/sentry/pull/90501/commits/82efb48e765ef6c102ba70be323f5334e50a2f56)